ouroboros-network/src/Ouroboros/Network/BlockFetch/Decision/BulkSync.hs

```diff
@@ -5,8 +5,86 @@
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 
--- | This module contains the part of the block fetch decisions process that is
--- specific to the bulk sync mode.
+-- | BulkSync decision logic
+--
+-- This module contains the part of the block fetch decisions process that is
+-- specific to the bulk sync mode. This logic reuses parts of the logic for the
+-- deadline mode, but it is inherently different.
+--
+-- Natural language specification
+-- ------------------------------
+--
+-- Definitions:
+--
+-- - Let @inflight :: peer -> Set blk@ be the outstanding blocks, those that
+--   have been requested and are expected to arrive but have not yet.
+--
+-- - Let @inflightIgnored :: Set blk@ be the outstanding blocks that have been
```

Comment:
`inflightIgnored` is updated below but never used for anything, therefore, why care about it?